pkg/operator/upgradebackupcontroller/upgradebackupcontroller_test.go

```diff
@@ -123,8 +123,8 @@ func Test_ensureRecentBackup(t *testing.T) {
 				Conditions: []configv1.ClusterOperatorStatusCondition{
 					{Type: backupConditionType, Reason: backupSuccess, Message: "UpgradeBackup pre 4.9 located at path "}},
 			},
-			wantBackupStatus: configv1.ConditionUnknown,
-			wantEventCount:   1, // pod created event
+			wantNilBackupCondition: true,
+			wantEventCount:         0, // pod created event
```

Comment:
`pod created event` doesn't make much sense with `wantEventCount: 0`, does it?

Reply:
+1 👍🏽